generators/entity/prompts.js

```diff
@@ -237,6 +238,45 @@ function askForRelationsToRemove() {
     }.bind(this));
 }
 
+function askForTableName() {
+    // don't prompt if there are no relationships
+    var entityTableName = this.entityTableName;
```

Comment:
If I add `this.log(entityTableName)` under that declaration, I see `undefined` if entityTableName is undeclared in the entity.json